### PR TITLE
First release

### PR DESCRIPTION
**Pull request by lats1**

### CODE REVIEW

File: src/Service/CprService.php [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+use ItkDev\Serviceplatformen\Service\PersonBaseDataExtendedService;
+
+class CprService
+{
+  private $personBaseDataExtendedService;
+
+  public function __construct()
+  {
+    $guzzleClient = \Drupal::httpClient();
+    $config = \Drupal::config('os2forms_cpr_lookup');
```
Comment (rimi-itk) [inline src/Service/CprService.php:21]: We should use dependency injection for this.

File: src/Element/CprElement.php [hunk 2]
```diff
@@ -0,0 +1,147 @@
+<?php
+
```
Comment (rimi-itk) [inline src/Element/CprElement.php:2]: We have too many blank lines here. We should set up some coding standards checks (cf. https://www.drupal.org/docs/contributed-modules/code-review-module/installing-coder-sniffer#s-composer-installer-plugin).

**Review by rimi-itk — CHANGES_REQUESTED** [review]
A few comments on coding standard and style added.

File: src/Service/CprService.php (hunk 1 above)
Comment (lats1) [reply src/Service/CprService.php:21]: Fixed in https://github.com/itk-dev/os2forms_cpr_lookup/pull/7

File: src/Element/CprElement.php (hunk 2 above)
Comment (lats1) [reply src/Element/CprElement.php:2]: Fixed in https://github.com/itk-dev/os2forms_cpr_lookup/pull/6

**Review by rimi-itk — APPROVED** [review]

**Pull request merged**